src/types/plugin-inputs.ts

```diff
@@ -49,6 +56,10 @@ export const userActivityWatcherSettingsSchema = T.Object({
   disqualification: thresholdType({
     default: "7 days",
   }),
+  /**
+   * List of events to consider as valid activity on a task
+   */
+  eventWhitelist: T.Array(T.String({ default: Object.values(eventWhitelist) })),
 });
```

Comment:
```suggestion
}, { default: {} }});
```
You will need this otherwise an empty configuration will fail.

Reply:
My mistake ty